apps/vs-code-designer/src/app/utils/codeless/updateBuildFile.ts

```diff
@@ -124,6 +124,32 @@ export function updateFunctionsSDKVersion(xmlBuildFile: Record<string, any>, dot
   return xmlBuildFile;
 }
 
+/**
+ * Allows local settings to be published to the directory by modifying the XML build file.
+ * @param context - The action context.
+ * @param xmlBuildFile - The XML build file to update.
+ * @returns The updated XML build file.
+ */
+export function allowLocalSettingsToPublishDirectory(context: IActionContext, xmlBuildFile: Record<string, any>): Record<string, any> {
+  try {
+    for (const itemGroup of xmlBuildFile['Project']['ItemGroup']) {
+      if ('None' in itemGroup) {
+        for (const item of itemGroup['None']) {
+          if (item['$']['Update'] === localSettingsFileName) {
```

Comment:
Are the `item['$']['Update']` references always going to be there or is there a chance they might be missing?  If they could be missing it might be best to optionally access them
ie. `item?.['$']?.['Update']`

Reply:
Yes, its expected that the schema remains the same, however it would be safer to add the optional access operator. I will add it, thanks!